pkg/limayaml/defaults.go

```diff
@@ -21,6 +21,13 @@ import (
 	"github.com/xorcare/pointer"
 )
 
+const (
+	DEFAULT_VIRTFS_SECURITY_MODEL   string = "mapped-xattr"
```

Comment:
```suggestion
	Default9pSecurityModel = "mapped-xattr"
```

Reply:
Will try to make some constructive split between parameters for the qemu `-virtfs` and parameters for the `mount -t 9p`, right now it is a bit of a mess